applications/isaac_holoscan_bridge/metadata.json

```diff
@@ -29,10 +29,119 @@
 			"Robotics"
 		],
 		"ranking": 3,
-		"dependencies": {},
-		"run": {
-			"command": "python3 <holohub_app_source>/app.py",
-			"workdir": "holohub_bin"
+		"dependencies": [
+			{
+				"id": "gpu",
+				"type": "hardware",
+				"name": "gpu",
+				"description": "NVIDIA GPU with at least 8GB memory required for Isaac Sim"
+			},
+			{
+				"id": "isaac_sim",
+				"type": "software",
+				"name": "Isaac Sim",
+				"description": "NVIDIA Isaac Sim simulation platform"
+			}
+		],
+		"modes": {
+			"interactive": {
+				"description": "Full Isaac Sim environment with interactive GUI and visualization for development, interactive debugging, and visual monitoring",
+				"run": {
+					"command": "python3 <holohub_app_source>/app.py",
+					"workdir": "holohub_bin",
+					"docker_args": [
+						"-u root",
+						"-e ACCEPT_EULA=Y",
+						"-e PRIVACY_CONSENT=Y",
+						"-v ${HOME}/docker/isaac-sim/cache/kit:/isaac-sim/kit/cache:rw",
+						"-v ${HOME}/docker/isaac-sim/cache/ov:/root/.cache/ov:rw",
+						"-v ${HOME}/docker/isaac-sim/cache/pip:/root/.cache/pip:rw",
+						"-v ${HOME}/docker/isaac-sim/cache/glcache:/root/.cache/nvidia/GLCache:rw",
+						"-v ${HOME}/docker/isaac-sim/cache/computecache:/root/.nv/ComputeCache:rw",
+						"-v ${HOME}/docker/isaac-sim/logs:/root/.nvidia-omniverse/logs:rw",
+						"-v ${HOME}/docker/isaac-sim/data:/root/.local/share/ov/data:rw",
+						"-v ${HOME}/docker/isaac-sim/documents:/root/Documents:rw"
+					],
+					"requirements": ["gpu", "isaac_sim"]
+				}
+			},
+			"headless": {
+				"description": "Headless Isaac Sim for automated workflows and CI/CD including automated testing, CI/CD pipelines, and batch processing",
+				"run": {
+					"command": "python3 <holohub_app_source>/app.py",
+					"workdir": "holohub_bin",
+					"run_args": ["--headless"],
```

Comment:
Can we merge `command` and `run_args` when under `modes` (keep them separate for backward compatibility under the top level `run`) ?